examples/parallel.py

```diff
@@ -46,6 +46,7 @@ def evaluate_metric(self, assignments):
   conn = Connection(client_token=client_token)
   experiment = conn.experiments().create(
     name="Parallel Test Franke Function",
+    project="sigopt-examples",
```

Comment:
typo?

Reply:
Or, since the two look different, is one a typo and one is not?

Reply:
yeah missing the `s` in the previous example